models/user.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+const {Model} = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+
+  User.init({
+    full_name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
```

Comment:
my friend (as a backend senior) suggests using validation from `express-validator` instead of Sequelize's built-in validator